maestro-cli/src/main/java/maestro/cli/runner/TestSuiteInteractor.kt

```diff
@@ -124,14 +129,77 @@ class TestSuiteInteractor(
         var flowStatus: FlowStatus
         var errorMessage: String? = null
 
+        // debug
+        val debug = FlowDebugMetadata()
+        val debugCommands = debug.commands
+        val debugScreenshots = debug.screenshots
+
+        fun takeDebugScreenshot(status: CommandStatus): File? {
+            val containsFailed = debugScreenshots.any { it.status == CommandStatus.FAILED }
```

Comment:
This method seems to be a duplicate of another`takeDebugScreenshot` in MaestroCommandRunner. Can we common this implementation?

Reply:
It's a convenience method for local references, doesn't make much sense to abstract it. The actual issue imo is that we have 2 difference entry points for running commands - but that's a problem for another day.